src/App.tsx

```diff
@@ -2,8 +2,19 @@ import { Provider } from "react-redux";
 import store from "./redux/store";
 import Todo from "./components/Todo";
 import { Toaster } from "./components/ui/sonner";
+import { useEffect } from "react";
 
 function App() {
+  const handleFetchTodo = async () => {
+    const response = await fetch("http://localhost:5173/todo.json").then(
```

Comment:
This is how we could fetch data from json by [fetch API ](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) . Please comment this before starting your implementation. This is just a example of fetching data